routemaster/state_machine/api.py

```diff
@@ -88,6 +88,7 @@ def create_label(app: App, label: LabelRef, metadata: Metadata) -> Metadata:
                 name=label.name,
                 state_machine=state_machine.name,
                 metadata=metadata,
+                updated=func.now(),
```

Comment:
Separate thought: does `func.now()` have a timezone? We seem to use timezone aware `datetime`s in some places and not in others; just want to make sure that we're not going to end up with weirdness as a result of slightly different sources of time.